pkgs/top-level/all-packages.nix

```diff
@@ -34647,4 +34647,6 @@ with pkgs;
   honeytail = callPackage ../servers/tracing/honeycomb/honeytail { };
 
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
+
+  thumbs = callPackage ../tools/misc/thumbs { };
```

Comment:
This file _should_ also be in alphabetical order. Place this line under the tools section in alphabetic order. It should be around 1080 (i think)